src/tespy/components/heat_exchangers.py

```diff
@@ -1263,6 +1283,27 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a parabolic trough.
+
+        Tamb : float
+            Ambient temperature Tamb / K.
+
+            .. math::
+
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \dot{Q}_{sol} \cdot \left(1 - \frac{4}
+                                                       {3} \frac{T_{amb}}
+                                                       {T_{sun}}\right)
+                Ex_{loss} = 0
```

Comment:
Same here with the linebreaks, also can you add the source for this equation in the .bib file and a citation here?